### ISSUE TITLE
Referencing channels from a multichannel audio file

### ISSUE THREAD

**Issue by mrlimbic** [bitwig/dawproject#59]
I'm considering adding support for this format.

However I can't see a way for a clip to reference channel 3 out of a 10 channel audio file.

Is this feature planned? 

It is pretty important for post production purposes as poly BWF wavs are the norm with different channels for different mics - i.e. 1-2: location mix, 3: boom, 4: lav actor 1, 5: lav actor 2 etc.

I could work around it by splitting all files into mono but I'd rather keep original files intact (with original metadata/timecode etc). Otherwise the format is very lossy when doing a round trip.

**Comment by kurasu** [comment]
It's not planned, and also not something we at bitwig would need anytime soon, but I'm open to ideas. The <Audio/> element would be the right place to include a channelMap (comma separated list?) attribute for this. 

What would your requirements be?

**Comment by mrlimbic** [comment]
Any number of sequential channels would cover almost every typical situation. It would be weird to need 3 & 7 as a stereo clip.

More likely example is 3,4 (rough stereo mix done on location) or just 7 single mic (mono) etc.

So start channel index + used channel count would do it. i.e start at channel 3, use 2 channels would end up meaning 3,4.

Any unusual situation could be covered by adding a channel remapping effect to the clip.

**Comment by kurasu** [comment]
You'd still need to say how many channels you want to output (mono/stereo/...), so the comma separated list is probably the best option. 

` <Audio channels="10" channelMap="3,4" duration="34.21" sampleRate="48000" id="id27">`

We can probably find a better name than channelMap though, as it doesn't imply the usage.

**Comment by mrlimbic** [comment]
Yes. A list of channels used would be fine too. That's how some other formats I deal with do it.
